roles/confluent.variables/defaults/main.yml

```diff
@@ -357,10 +371,10 @@ schema_registry_custom_properties: "{{ schema_registry.properties }}"
 
 #### Kafka Rest Variables ####
 
-### Only use to customize Linux User Rest Proxy Service runs with. User must exist on host.
+### Only use to customize Linux User Rest Proxy Service runs with. Default user is cp-kafka-rest
 kafka_rest_user: "{{kafka_rest_default_user}}"
 
-### Only use to customize Linux Group Rest Proxy Service user belongs to. Group must exist on host.
+### Only use to customize Linux Group Rest Proxy Service user belongs to. Default group is confluent
```

Comment:
Modify this variable to customize the Linux Group that the Rest Proxy Service user belongs to. Default group is confluent.